src/main/archetype/pom.xml

```diff
@@ -44,8 +44,8 @@
  #if ( $includeFormsheadless == "y" )
         <module>ui.frontend.react.forms.af</module>
  #end
-        <module>ui.apps</module>
         <module>ui.apps.structure</module>
+        <module>ui.apps</module>
```

Comment:
Just making .structure an explicit dependency of ui.apps is more robust and allows Maven to calculate the order correctly.